modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/tcs/TcsControllerEpics.scala

```diff
@@ -369,8 +370,8 @@ object TcsControllerEpics extends TcsController {
 
   private def setM2Guide(c: M2GuideConfig): SeqAction[Unit] = TcsEpics.instance.m2GuideCmd.setState(encode(c))
 
-  private val tcsTimeout = Seconds(30)
-  private val agTimeout = Seconds(30)
+  private val tcsTimeout = Seconds(60)
```

Comment:
I guess this will apply to all instruments, Why was it changed?

Reply:
It was changed because I just found today that the science fold can take 45[s] to reach a position. That affects also `tcsTimeout`.